Jenkinsfile

```diff
@@ -1,9 +1,14 @@
 pipeline {
     agent {
         docker {
-            image 'python:3.7-stretch'
+            image 'teambitflow/python-docker:3.7-stretch'
+            args '-v /var/run/docker.sock:/var/run/docker.sock'
         }
     }
+    environment {
+        registry = 'teambitflow/python-bitflow'
+        registryCredential = 'dockerhub'
+    }
     stages {
```

Comment:
Please start with the "Git" stage (see bitflow4j) - otherwise the git committer email is unknown